Flat/src/ModulesJets.cc

```diff
@@ -349,7 +354,7 @@ void PandaAnalyzer::JetHbbReco()
   if (centralJets.size() > 1) {
     vector<Jet*> csvSortedJets = centralJets;
     sort(csvSortedJets.begin(), csvSortedJets.end(),
-        [](panda::Jet *x, panda::Jet *y) -> bool { return x->csv > y->csv; });
+        [](panda::Jet *x, panda::Jet *y) -> bool { return x->cmva > y->cmva; });
```

Comment:
Again, no. Hardcoding changes like this messes up other analyses. Make it configurable, only for h->bb.

Reply:
Ok, already have "useCMVA" flag, just need to add it here